packages/terra-networks/lib/index.ts

```diff
@@ -8,6 +8,7 @@ export interface TerraNetwork extends Network {
   chainID: string
   asset: string
   codeId: number
+  tokenAddress?: string
```

Comment:
This feels like the wrong way of constructing this.. I mean the token address is not described by a network
To be honest, i think it should be same for asset, These should rather be constructor parameters to the providers